src/JBSucker.sol

```diff
@@ -117,6 +112,12 @@ abstract contract JBSucker is JBPermissioned, ERC165, IJBSucker {
     // -------------------- internal stored properties ------------------- //
     //*********************************************************************//
 
+    /// @notice The peer sucker on the remote chain.
+    address private remotePeer;
+
+    /// @notice The ID of the project (on the local chain) that this sucker is associated with.
+    uint256 private localProjectId;
```

Comment:
nit: alphebetize prop. 
why "private" and not "internal"?

Reply:
We might want to store them some other way thats more optimized in the future, doing it this way forces sucker implementations to use the getter method which makes changes in the future easier.